tests/Frontend/PathSpec.hs

```diff
@@ -33,6 +35,19 @@ import Network.Wai
 import Test.Hspec.Wai (get, post, shouldRespondWith)
 import qualified Test.Hspec.Wai.QuickCheck as Wai (property)
 
+isBrokenPath :: Main -> Bool
```

Comment:
For the report handler, I fetched the comment from the DB, since the path has been randomly generated so the different parts are not tied together. This should be fixed once we have a complete comment id.